compiler/src/Parse/Shader.hs

```diff
@@ -213,6 +213,9 @@ eatSomethingElse openCurly fp offset terminal row col =
     else if word == 0x2F {- / -} then
       eatComment (eatSomethingElse openCurly) fp offset terminal row col
 
+    else if word == 0x23 {- # -} then
+      eatLineComment (eatSomethingElse openCurly) fp (offset + 1) terminal row (col + 1)
```

Comment:
Not sure if this is needed, because `#`  will not start in-between expressions or declarations.